# Binary dump from Oculus sonar for post-processing

Header header            # Standard header

uint8[] data             # Raw dump of Oculus data
